paasta_tools/cli/cmds/secret.py

```diff
@@ -138,6 +138,8 @@ def get_plaintext_input(args):
                 break
             lines.append(line)
         plaintext = '\n'.join(lines).encode('utf-8')
+        print("The secret as a Python string is:", repr(plaintext))
+        print("Please make sure this is correct.")
```

Comment:
nit: maybe you can give slightly different wording with a little more context like `"Make sure this matches what you typed in and that there are no truncations or encoding issues"`?

just a thought, feel free to ignore.